src/Resources/skeleton/event/Listener.tpl.php

```diff
@@ -0,0 +1,14 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace; ?>;
+
+<?= $use_statements; ?>
+
+final class <?= $class_name ?>
```

Comment:
Add a `\n` on the end - see https://github.com/symfony/maker-bundle/blob/main/src/Resources/skeleton/message/Message.tpl.php#L5

If `?>` is the last thing on a line, the PHP sucks up the whitespace before the next char. Not sure why we're seeing this on Windows only, but I bet it'll fix it (maybe this odd behavior is windows-only)

Reply:
It's okay about my generated files in appveyor, thanks @weaverryan :wink: 

The two errors left are about `TemplateLinterTest`